ui/component/copyableStreamkeyUnmask/view.jsx

```diff
@@ -0,0 +1,41 @@
+// @flow
+import Button from 'component/button';
+import React from 'react';
+
+type Props = {
+  snackMessage: ?string,
+  doToast: ({ message: string }) => void,
+  label?: string,
+  primaryButton?: boolean,
+  name?: string,
+};
+
+export default function CopyableText(props: Props) {
+  const { doToast, primaryButton = false } = props;
+
+  function showStreamkey() {
+    if (document.getElementById('livestream-key').type === 'password') {
+      doToast({ message: __('Showing Stream Key') });
+      document.getElementById('livestream-key').setAttribute('type', 'text');
+      document.getElementById('streamkey-unmask').style.color = 'white';
+      document.getElementById('streamkey-unmask').innerText = 'Hide Stream key';
+    } else if (document.getElementById('livestream-key').type === 'text') {
+      doToast({ message: __('Hiding Stream Key') });
+      document.getElementById('livestream-key').setAttribute('type', 'password');
+      document.getElementById('streamkey-unmask').style.color = 'white';
+      document.getElementById('streamkey-unmask').innerText = 'Show Stream key';
+    }
+  }
+
+  return (
+    <Button
+      button={primaryButton ? 'primary' : 'secondary'}
+      label={__('Show Stream key')}
+      id="streamkey-unmask"
+      name="streamkey-unmask"
+      onClick={() => {
+        showStreamkey();
+      }}
+    />
```

Comment:
As mentioned in a prior comment, a separate component is not necessary.
On top of that, the button location is not ideal, and there is no gap between components.  Here's one suggestion:

![Untitled](https://user-images.githubusercontent.com/64950861/134123099-7795a82f-63ad-4050-bb7a-75776b794681.png)

Reply:
I'm not that great with React so I'm not able to replicate that.